receiver/couchdbreceiver/factory.go

```diff
@@ -26,14 +26,14 @@ func NewFactory() receiver.Factory {
 }
 
 func createDefaultConfig() component.Config {
+	clientConfig := confighttp.NewDefaultClientConfig()
+	clientConfig.TLSSetting = configtls.ClientConfig{}
```

Comment:
```suggestion
	clientConfig.TLSSetting = configtls.NewDefaultClientConfig()
```
Should we update this reference too it's the same underlying concern?

Reply:
These changes are for `confighttp`. I can create another issue for `configtls` similarely to #35457 ?

Reply:
Sure, another issue would probably be good. It looks like the goal is to move all configs over the the `NewDefault...` initialization, as noted [here](https://github.com/open-telemetry/opentelemetry-collector/issues/9478#issuecomment-1930475996), but I understand it's unrelated to this PR so we don't have to do it yet. 👍

Reply:
Opened https://github.com/open-telemetry/opentelemetry-collector/issues/11383